cmd/common.go

```diff
@@ -126,16 +127,24 @@ func NewConfig(paths []string, extras []string) types.PlatformConfig {
 	return base
 }
 
-func mergeConfigs(base *types.PlatformConfig, paths []string) error {
+func mergeConfigs(base *types.PlatformConfig, paths []string, encrypted bool) error {
```

Comment:
I think we should split this into 2 methods e.g. mergeSops|Encrypted and mergePlaintext  - Boolean parameters often make it difficult to read

Reply:
I originally started with that approach.  Apart from the small section where the bool is checked, the methods would be identical and then contain a lot of duplication.  Trying to set it up such both merge sops and merge plain both call a mergebytes function was making the recursion awkward because the cfg object would need to be declared at the wrong scope where the path is still known.  But I can try again to see if I can make that work a bit better